metals/src/main/scala/scala/meta/internal/bsp/BspServers.scala

```diff
@@ -229,6 +229,10 @@ object BspServers {
         None
       },
       details => {
+        if (details.getVersion() == null) {
+          val json = ujson.read(text)
+          json("millVersion").strOpt.map(details.setVersion)
```

Comment:
Unfortunately it's called `millVersion` instead of `version` in mill bsp config.